content/en/bits_ai/query_examples.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Example Natural Language Queries
+kind: guide
+disable_toc: false
+private: true
+is_beta: true
+---
+
+{{< whatsnext desc="For instructions on running natural language queries in Datadog, see:">}}
+    {{< nextlink href="bits_ai/getting_started/#in-the-chat-panel" >}}Querying in the chat panel{{< /nextlink >}}
+    {{< nextlink href="bits_ai/getting_started/#in-a-search-bar" >}}Querying in a search bar{{< /nextlink >}}
+{{< /whatsnext >}}
+
+## Logs
+You can query your logs in natural language from the Logs Explorer search bar.
+
+- "Find errors in AWS CloudTrail where a user is assuming a different user's role"
+{{< img src="bits_ai/cloudtrail-user-role-errors.png" alt="Query result for user role changes" style="width:90%;">}}
+- "Create a pie chart of error logs by service"
+{{< img src="bits_ai/logs-pie-chart.png" alt="Query result for a pie chart of error logs by service" style="width:90%;">}}
+- "Show me patterns of errors for users checking out"
+{{< img src="bits_ai/checkout-error-patterns.png" alt="Query result for user checkout errors" style="width:90%;">}}
+
+## APM traces
+
+To query APM traces, use the chat panel.
+
+Examples:
+- "Show me traces for web-store that are slower than 1s"
+{{< img src="bits_ai/slow-web-store-traces.png" alt="Query result for slow traces" style="width:90%;">}}
+
+## Infrastructure data (Inventories SQL)
+
+Infrastructure resource data can be queried in natural language from the search bar of [Inventories SQL][3], a centralized interface for infrastructure resource data. You can also use the chat panel to query infrastructure data from anywhere in Datadog.
```

Comment:
This 404d for me in my sandbox. Maybe a banner here mentioning that Inventories is in beta?